server.js

```diff
@@ -115,6 +116,9 @@ const parseQueryString = (q, body, options = {}) => {
   if (options.profile) {
     applyProfileOptions(options);
   }
+  if (options.fetchLicense) {
```

Comment:
Since node is a single process event driven model, one request will affect multiple responses. We need to refactor the codebase and make the fetch license operate with an options argument instead of only relying on env variables